docs/src/components/Markdown.css

```diff
@@ -7,3 +7,7 @@
   margin-bottom: 8px;
   margin-top: 8px;
 }
+
+.Markdown pre {
```

Comment:
what's this?

Reply:
@AlbertCarreras on smaller screens, we were seeing a horizontal scrollbar on the whole page because of the code we added to showcase the gestalt datepicker.

```
![image](https://user-images.githubusercontent.com/127199/85160149-4893f400-b213-11ea-8835-cab9195d7853.png)
```